contracts/p1/Distributor.sol

```diff
@@ -201,6 +233,33 @@ contract DistributorP1 is ComponentP1, IDistributor {
         require(rTokenDist != 0 || rsrDist != 0, "no distribution defined");
     }
 
+    function _validateDistributionTable() internal view {
+        uint256 dLen = destinations.length();
+        DAOFeeRegistry daoFeeRegistry = main.daoFeeRegistry();
+        require(destinations.length() > 0, "no destinations");
+
+        RevenueTotals memory revTotals;
+        uint24 daoTotals;
+
+        for (uint256 i = 0; i < dLen; ++i) {
+            address dest = destinations.at(i);
+            RevenueShare storage share = distribution[dest];
+            revTotals.rTokenTotal += share.rTokenDist;
+            revTotals.rsrTotal += share.rsrDist;
+
+            if (dest == DAO_FEE) {
+                daoTotals += share.rsrDist;
```

Comment:
i think this is still an open question. would be better to just allow veRSR to set the DAO distribution entirely